ui/StatusQ/src/typesregistration.cpp

```diff
@@ -113,6 +113,14 @@ void registerStatusQTypes() {
                                                   return new OnboardingEnums;
                                               });
 
+// create import alias from QtGraphicalEffects 1.15 to Qt5Compat.GraphicalEffects
+// in order to satisfy both Qt 5/6, relying the same version of qml code
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    qmlRegisterModule("QtGraphicalEffects", 1, 15);
```

Comment:
Hmm, do you need this line at all?

Reply:
Yes, otherwise `import QtGraphicalEffects 1.15` from qml will result with module not found.